source/docx_generator/docx_generator.py

```diff
@@ -74,7 +76,7 @@ def _set_jinja2_custom_environment(self, base_path: str, template: DocxTemplate,
         jinja2_custom_globals = Globals(base_path, template, jinja2_environment)
 
         jinja2_custom_filters.set_available_filters()
-        jinja2_custom_globals.set_available_globals()
+        jinja2_custom_globals.set_available_globals(self._image_handler)
```

Comment:
Shouldn't it be the following ? `set_available_globals` doesn't accept any arguments https://github.com/dfir-iris/docx-generator/blob/1c5a841e9c23b9f1bd66fab44871f578da067468/source/docx_generator/globals/globals.py#L55 

```suggestion
        jinja2_custom_globals.set_available_globals()
```

Reply:
Oh damn... This was from an old version.
Had some troubles setting up iris in a docker-dev-environment with docx-generator as a source-module. Therefore I was copy over the changes manually.
Changed it in https://github.com/dfir-iris/docx-generator/pull/1/commits/f106c4b3dd9978780ad474f287b60c84c24ea0c7